app/src/main/java/com/example/android/example/TrailerViewHolder.java

```diff
@@ -0,0 +1,59 @@
+package com.example.android.example;
+
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.android.example.Data.Trailer;
+import com.squareup.picasso.Picasso;
+
+import java.util.List;
+
+import static com.example.android.example.Adapters.TrailerAdapter.YT_BASE_PATH;
```

Comment:
avoid static imports

Reply:
You can read about it here and issues as well https://en.wikipedia.org/wiki/Static_import